initramfs/init.sh

```diff
@@ -21,7 +21,7 @@ rwmode=""
 
```

Comment:
The #!/bin/busybox sh is not recognized error we can solve by using #!/bin/sh. I wanted to be super explicit, but we also create the sh symlink from the initramfs.lst anyway.